<launch>
	<param name="limb" value="right">

	<node name="joint_action_server" pkg="cs4752_proj3" type="joint_action_server.py" output="screen"></node>
	<node name="averager" pkg="cs4752_proj3" type="averager.py" output="screen"></node>
	<node ns="kinect" name="vision" pkg="cs4752_proj3" type="vision.py" output="screen"></node>
	<node name="get_blocker_offset" pkg="cs4752_proj3" type="get_blocker_offset.py"></node>
	<node name="do_everything.py" pkg="cs4752_proj3" type="do_everything.py"></node>
    <node unless="$(arg calibrate)" pkg="tf" type="static_transform_publisher" name="kinect_tf" args="0.03 0.42 1.40 0.706757866675 -0.706757866675 0.0222107607464 0.0222107607464 /camera_rgb_optical_frame /base 300"/>
